src/transformers/ava.js

```diff
@@ -17,6 +17,7 @@ import proxyquireTransformer from '../utils/proxyquire';
 
 const SPECIAL_THROWS_CASE = '(special throws case)';
 const SPECIAL_BOOL = '(special bool case)';
+const SPECIAL_ASSERTION_CASE = '(special assertion case)';
```

Comment:
Small thing: but as everything is "assertions" can we call it `SPECIAL_PLAN_CASE`. That might be a bit easier to remember.